custom-elements/upgrading.html

```diff
@@ -185,6 +185,32 @@
 
 });
 
+test(() => {
+    class InternalsDisabledElement extends HTMLElement {
+      static get disabledFeatures() { return ['internals']; }
+    }
+    let element = document.createElement('internals-disabled');
+    customElements.define('internals-disabled', InternalsDisabledElement);
+    element.attachInternals();
```

Comment:
Per step 4 of https://html.spec.whatwg.org/multipage/custom-elements.html#dom-attachinternals, attachInternals() should throw `NotSupportedError`, so it looks like this is an invalid case?

Reply:
Oh, you're right.  We can't do attachInternals() successfully before upgrading.
Removed the test.